src/app/services/room.service.ts

```diff
@@ -46,11 +63,12 @@ export class RoomService {
   mockPriorities: number[] = [1, 2, 3, 4];
 
   base = 'http://plpoker-api.azurewebsites.net';
+
   constructor(private http: HttpClient) {
   }
 
   createRoom(roomName: string) {
-    return this.http.put(this.base+'/api/Rooms',{name:roomName});
+    return this.http.put(this.base + '/api/Rooms', {name: roomName});
```

Comment:
It's a good practice in JS to use literal templates instead of string concatenations. It would be like:
${this.base}/api/Rooms written in between backticks

Reply:
yea, you have right. I always forget about this